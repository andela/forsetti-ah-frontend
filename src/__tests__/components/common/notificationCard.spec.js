import React from 'react';
import { shallow } from 'enzyme';
import NotificationCard from '../../../components/common/noficationCard';

describe('Notification Card', () => {
  it('should render NotificationCard correctly', () => {
      const wrapper = shallow(<NotificationCard />);
      expect(wrapper).toMatchSnapshot();
  });

  it('should render NotificationCard with article type props', () => {
    const notices = {
        type: 'article'
    }
    const wrapper = shallow(<NotificationCard {...notices} />);
    expect(wrapper).toMatchSnapshot();
  });
  it('should render NotificationCard with comment type props', () => {
    const notices = {
        type: 'comment'
    }
    const wrapper = shallow(<NotificationCard {...notices} />);
    expect(wrapper).toMatchSnapshot();
  });
});
