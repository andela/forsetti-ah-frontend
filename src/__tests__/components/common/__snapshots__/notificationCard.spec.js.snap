// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification Card should render NotificationCard correctly 1`] = `ShallowWrapper {}`;

exports[`Notification Card should render NotificationCard with article type props 1`] = `ShallowWrapper {}`;

exports[`Notification Card should render NotificationCard with comment type props 1`] = `ShallowWrapper {}`;
